router/io-spec.gobra

```diff
@@ -472,7 +472,7 @@ requires unfolding acc(sl.AbsSlice_Bytes(raw, 0, len(raw)), _) in
     0 < metaHdr.SegLen[0] &&
     base.ValidCurrInfSpec() &&
     base.ValidCurrHfSpec() &&
-    len(raw) == pktLen(seg1, seg2, seg3)
+    len(raw) >= pktLen(seg1, seg2, seg3)
```

Comment:
This is necessary because `ReadBatch()` returns byte arrays longer than the actual message. If I would want to keep the `==` I would have to `unfold AbsSlice_Bytes(msg, 0, len(msg))`and `fold AbsSlice_Bytes(msg, 0, len(msg[:msg.N))`which I cannot do in a postcondition. Logically I don't see a problem if I change `==`to `>=`. Do you think this could cause performance problems?

Reply:
no, this should be fine. I just wanted to make sure that there was a reason to make the spec "looser"